packages/common/src/content/edit.ts

```diff
@@ -168,6 +169,12 @@ export async function updateContent(
     }
   }
 
+  if (content.schedule.mode === "automatic") {
+    await deleteSchedule(item, requestOptions);
```

Comment:
should we first fetch the schedule, and then only delete the schedule when one exists and the mode is automatic?